src/deployment/azuredeploy.json

```diff
@@ -101,6 +102,35 @@
             "apiVersion": "2018-11-30",
             "location": "[resourceGroup().location]"
         },
+        {
+            "type": "Microsoft.KeyVault/vaults",
+            "apiVersion": "2016-10-01",
+            "name": "[variables('keyVaultName')]",
+            "location": "[resourceGroup().location]",
+            "properties": {
+                "enabledForDeployment": true,
+                "enabledForDiskEncryption": false,
+                "enabledForTemplateDeployment": true,
+                "tenantId": "[subscription().tenantId]",
+                "accessPolicies": [
+                     {
+                        "objectId": "[reference(resourceId('Microsoft.Web/sites', parameters('name')), '2019-08-01', 'full').identity.principalId]",
+                        "tenantId": "[subscription().tenantId]",
+                        "permissions": {
+                            "secrets": ["get", "list", "set", "delete"]                        
+                        }
+                    }
+                ],
+                "sku": {
+                    "name": "standard",
+                    "family": "A"
+                },
+                "networkAcls": {
```

Comment:
Does this property actually exist in the ARM resource API version we are using? 

Compare: https://docs.microsoft.com/en-us/azure/templates/microsoft.keyvault/2016-10-01/vaults#vaultproperties-object

Vs: https://docs.microsoft.com/en-us/azure/templates/microsoft.keyvault/2018-02-14/vaults#vaultproperties-object

Reply:
updated the version to the latest which has this property